internal/client/grpc/things/thingsclient.go

```diff
@@ -0,0 +1,40 @@
+package things
+
+import (
+	"io"
+	"io/ioutil"
+
+	"github.com/mainflux/mainflux/pkg/errors"
+	gogrpc "google.golang.org/grpc"
+
+	"github.com/mainflux/mainflux"
+	thingsapi "github.com/mainflux/mainflux/things/api/auth/grpc"
+
+	grpcClient "github.com/mainflux/mainflux/internal/client/grpc"
+
+	"github.com/mainflux/mainflux/internal/env"
+)
+
+const envThingsAuthGrpcPrefix = "MF_THINGS_AUTH_GRPC_"
+
+var (
+	errGrpcConfig = errors.New("failed to load grpc configuration")
+)
+
+func Setup(envPrefix, jaegerURL string) (mainflux.ThingsServiceClient, *gogrpc.ClientConn, io.Closer, string, error) {
```

Comment:
Avoid returning the log message as a return value, but pass loger to the function instead if needed.

Reply:
Actually, I don't want to pass logger to them, Since they are internal package and they are library. 
And the `main.go` can handle the  logger. 
 I will change it bool value